src/main/scala/microsites/MicrositesPlugin.scala

```diff
@@ -79,7 +79,14 @@ object MicrositesPlugin extends AutoPlugin {
                             "gray-lighter"    -> "#F4F3F4",
                             "white-color"     -> "#FFFFFF"),
     micrositeGithubOwner := "47deg",
-    micrositeGithubRepo := "sbt-microsites")
+    micrositeGithubRepo := "sbt-microsites",
+    micrositeKazariStyle := "kazari-style-dark",
+    micrositeKazariEvaluatorUrl := "https://scala-evaluator-212.herokuapp.com",
+    micrositeKazariEvaluatorToken := "",
```

Comment:
If we are going to leave the scala-evaluator as default, how can people get the evaluator token?

CC @raulraja